codegen/src/main/java/software/amazon/awssdk/codegen/internal/ImmutableMapParameter.java

```diff
@@ -1,304 +0,0 @@
-/*
```

Comment:
this class duplicates with `software.amazon.awssdk.core.util.ImmutableMapParameter`.